### PR TITLE
Ingestion

### PR DESCRIPTION
**Pull request by zyb2n**
merge existing ingestion branch

### CODE REVIEW

File: Ingestion/fargate-login.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -x
```
Comment (jjoyce0510) [inline Ingestion/fargate-login.sh:2]: So this gets us into the ECS container?

File: Ingestion/templates/python.ecs.template.yaml [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    Properties:
+      RoleName: !Sub "${AWS::StackName}-task-role"
+      #RoleName: !Sub "datahub-ingestion-task-role"
+      Policies:
+        - PolicyName: ssmmessages-policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+            - Action: 
+                - ssmmessages:CreateControlChannel
```
Comment (jjoyce0510) [inline Ingestion/templates/python.ecs.template.yaml:184]: Why are these priveleges required again?

Comment (zyb2n) [reply Ingestion/templates/python.ecs.template.yaml:184]: needed for fargate login

File: Ingestion/fargate-login.sh (hunk 1 above)
Comment (zyb2n) [reply Ingestion/fargate-login.sh:2]: use sample ./fargate-login.sh to login to container, for example "./fargate-login.sh dev-ingestion"

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**